docs-chef-io/content/inspec/resources/aws_sqs_queue.md

```diff
@@ -122,20 +127,62 @@ end
 
 The control will pass if the describe returns at least one result.
 
-Use `should_not` to test the entity should not exist.
+Use `should` to test the entity should exist.
 
 ```ruby
 describe aws_sqs_queue('https://sqs.ap-southeast-2.amazonaws.com/1212121/MyQueue') do
   it { should exist }
 end
 ```
 
+Use `should_not` to test the entity should not exist.
+
 ```ruby
 describe aws_sqs_queue('https://sqs.ap-southeast-2.amazonaws.com/1212121/MyQueueWhichDoesntExist') do
   it { should_not exist }
 end
 ```
 
+### policy_statement_principal_all_permissions_enabled
+
+The control will pass if the describe returns at least one result.
+
+Use `should` to test the entity should exist.
```

Comment:
This should explain what the result is. Maybe:

```suggestion
The control will pass if at least one Principal has all permissions enabled.

Use `should` to test when a Principal with all permissions should exist.
```

Or something like that. I'm not sure of the context here.